test/unit/anchor.js

```diff
@@ -72,6 +75,69 @@ describe('Anchor', function() {
           ref : 'ten'
         }, 'name', points).should.throw()
     })
+    it('aggregate_intersect', function() {
```

Comment:
Just a reminder that the backward projection would also need a test here somewhere. Also, I try not to have underscores in the test names for a more uniform and non-code aesthetic. Could you please rename this to just `intersect`?

Reply:
Noted!

Reply:
Re. the backward projection, see my comment on `src/anchor.js`.

Reply:
Added test cases to account for intersection on the negative Y axis.